impeller/core/resource_binder.cc

```diff
@@ -0,0 +1,11 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/core/resource_binder.h"
+
+namespace impeller {
+
+//
```

Comment:
What's this?

Reply:
This is how all the other otherwise header-only TUs are handled in impeller